testutils/src/main/java/com/mparticle/networking/MockServer.java

```diff
@@ -419,16 +419,16 @@ private MPUrl getUrl(String identityPath, Long mpid) {
     private MParticleBaseClientImpl getClient() {
         ConfigManager configManager = null;
```

Comment:
i think we can delete this ConfigManger variable if it's not being used

Reply:
👍